### DISCUSSION TITLE
connecting two JK-BMS - parallel mode

### DISCUSSION THREAD

**Discussion by studiowena** [Sleeper85/esphome-jk-bms-can#51] [Ideas]
Hi,

It would be nice we could use two JK-BMS in parallel mode when we have 2x16 cells batteries in parallel

**Comment by Sleeper85** [comment]
Hi,

The new multi-bms version is ready to test.

https://github.com/Sleeper85/esphome-jk-bms-can/tree/development

😉

**Comment by krbo87** [comment]
Hi,

How to get access to the development folder?

Den ons. 22. maj 2024 kl. 14.01 skrev Sleeper85 ***@***.***>:

> Hi,
>
> The new multi-bms version is ready to test.
>
> https://github.com/Sleeper85/esphome-jk-bms-can/tree/development
>
> 😉
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Sleeper85/esphome-jk-bms-can/discussions/51#discussioncomment-9520719>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A5FERA74GHGDSCHAB46BDM3ZDSCLBAVCNFSM6AAAAABIDPJ5GWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TKMRQG4YTS>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: <Sleeper85/esphome-jk-bms-can/repo-discussions/51/comments/9520719@
> github.com>
>


-- 
MvH
Kristian Overbeck
Søndergyden 29
6640 Lunderskov
Tlf. 23403980

**Comment by Sleeper85** [comment]
You don't have access to the development branch?

You just need to download the files. The most important being one of the main YAML, the secrets.yaml that you must configure and the packages folder.

**Comment by Sleeper85** [comment]
Except for the new JK-BMS and for those with a PVbrain2 board. This multi-BMS version works with a single ESP32 and maximum 3x UART BMS and/or 3x BLE BMS. A more powerful ESP32-S3 is better.
